main.js

```diff
@@ -0,0 +1,156 @@
+$(document).ready(function(){
+
+    $('#square').hide();
+
+    //$( "*", document.body ).click(function( event ) {
+    //    var offset = $( this ).offset();
+    //    event.stopPropagation();
+    //    $( "#result" ).text( this.tagName +
+    //        " coords ( " + offset.left + ", " + offset.top + " )" );
+    //});
+
+    function render(val){
+        console.log(val);
+        $('#square').css({
+            "width" : val.width,
+            "height": val.height,
+            "background" : val.background,
+            "left": val.left + "%",
+            "top": val.top + "%",
+            "textAlign": val.textAlign,
+            "border": val.border,
+            "marginLeft": val.marginLeft,
+            "marginTop": val.marginTop,
+            "position": val.position
+        });
+    }
+
+    function defaultSquareValues(){
+        $('#square').css({
+            "width" : 300 + "px",
+            "height": 300 + "px",
+            "background": "red",
+            "left": 50 + "%",
+            "top": 50 + "%",
+            "textAlign": "center",
+            "border": 3 + "px solid black",
+            "marginLeft": -150 + "px",
+            "marginTop": -150 + "px",
+            "position": "absolute"
+        })
+    }
+
+
+    function error(){
+        $('#h1').html("Fatal Error! 404 - API Not Reachable!");
+    }
+
+    function success(){
+        alert("API Not Reachable! Using default square values!");
+    }
+
+
+    $('#btn').on('click', function(){
+
+        $.ajax({
+            url: 'http://demo6147751.mockable.io/ninoslav',
+            method: 'GET',
+            dataType: 'json',
+            success: function(data){
+                render(data);
+
+                    $('#btn').hide();
+                    $('#square').show();
+
+            },
+            error: function(){
+                    success();
+                    defaultSquareValues();
+
+                    $('#btn').hide();
+                    $('#square').show();
+            }
+
+        })
+    });
+
+    $(document).keydown(function(e){
+
+        var offset = $( "#square" ).offset();
+        var poz  = $("#square").position();
+        console.log(poz);
+
```

Comment:
switch case logic needs to be pulled out in to functions. One function if possible